common/lib/common-utils/pnpm-lock.yaml

```diff
@@ -35,7 +35,7 @@ importers:
       concurrently: ^6.2.0
       copyfiles: ^2.4.1
       cross-env: ^7.0.3
-      eslint: ~8.6.0
+      eslint: ~8.50.0
```

Comment:
Something's funky here, the specifier in package.json is ~8.55.0, it can't have resolved 8.50, unless this was from a previous resolution and the lockfile needs to be updated.